github.go

```diff
@@ -164,15 +164,37 @@ func getCommits(issueable Issueable, pullRequests PullRequests) ([]*github.Repos
 			message := fmt.Sprintf("Getting commits for PR %s failed", issue.FullName())
 			return nil, &ErrorResponse{err, http.StatusBadGateway, message}
 		}
+		isLastPage := resp.NextPage == 0
+		if isLastPage && !isExpectedHead(*lastCommit(pageCommits).SHA) {
```

Comment:
I want to keep the if's short-circuit behavior: https://github.com/salemove/github-review-helper/pull/29/commits/7cae0a9b2c42ef86be9c9524911677dca78025da